src/twisted/python/_setup.py

```diff
@@ -96,6 +96,7 @@
         'pyasn1',
         'cryptography >= 0.9.1',
         'appdirs >= 1.4.0',
+        'pynacl >= 1.0.1',
```

Comment:
The question is really: should Twisted be concerned about environments where someone will say "uh oh, pynacl, you can't install that, denied"?  Or is this the kind of thing that will be configured at a policy level?  (After all you'd need to actually use these ciphers in your deployment to exercise this code.)